### DISCUSSION TITLE
Error during ./ww3_ounf

### DISCUSSION THREAD

**Discussion by sohan-pm** [NOAA-EMC/WW3#1177] [Q&A]
*** WAVEWATCH III WERROR IN W3EXNC : 
     OUT OF RANGE REQUEST FOR NBIPART = 2
     MAX SWELL PARTITIONS (NOSW) = 1

**Comment by mickaelaccensi** [comment]
the parameters requested in postprocessing (ww3_ounf, ww3_ounp) must be consistent with the list provided in ww3_shel or ww3_multi.
